source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,49 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.39 Aspect Ratios
+
+[Aspect Ratios Vuetify documentation page](https://vuetifyjs.com/en/components/aspect-ratios/)
+
+```java
+    @Test
+    public void aspectRatioImageTests() {
+        aspectRatiosTestsData.aspectRatiosTestData().forEach(
+                dataObject -> {
+                    getDriver().manage().window().setSize(new Dimension(dataObject.getWidth(),
+                        dataObject.getHeight()));
+                    aspectRatiosContainer.has().ratio(16, 9);
+                }
+        );
+    }
+```
+- __Java__: _com.epam.jdi.light.vuetify.elements.common.AspectRatios.java_
+
+Aspect Ratios - the `v-responsive` component can be used to fix any section to a specific aspect ratio.
+
+![Aspect Ratios Example](../../images/vuetify/aspect_ratios.png)
+
+
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <div>
+    <v-card>
+      <v-responsive :aspect-ratio="16/9">
+        <v-card-text>
+          This card will always be 16:9 (unless you put more stuff in it)
+        </v-card-text>
+      </v-responsive>
+    </v-card>
+  </div>
+</template>
+```
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 

![image](https://user-images.githubusercontent.com/55189259/212975164-d661b64c-76cf-4de9-b8db-d8be1582451a.png)